examples/fluids/navierstokes.h

```diff
@@ -299,6 +299,7 @@ struct BlasiusContext_ {
   CeedScalar Uinf;      // !< Velocity at boundary layer edge
   CeedScalar P0;        // !< Pressure at outflow
   CeedScalar theta0;    // !< Temperature at inflow
+  CeedInt weakT;        // !< flag to set Temperature at inflow
```

Comment:
Should these be `bool` instead of `int`?

```suggestion
  bool weakT;           // !< flag to set Temperature at inflow
```
(also includes changing to `PetscOptionsInt` -> `PetscOptionsBool`, and `if(weakT == 0)` to `if(weakT)`)

Reply:
Done, thanks.